en/docs/references/analyze-performance.md

```diff
@@ -0,0 +1,58 @@
+# Performance Analysis
+
+The performance Analyzer tool is an AI-based tool that generates intelligent estimations of the performance of service-based applications based on historical data. To generate the forecasts, it uses advanced Machine Learning techniques.
+
+Making accurate performance estimations before releasing an application you developed is important due to the following reasons:
+
+- When you create Service-level Agreements (SLAs) for your application, an accurate forecast of performance allows you to provide valuable insights relating to the scalability of the application and the ways to optimize its performance.
+- You can avoid missing important performance characteristics as a result of combining multiple applications including external ones.
+- You can correctly assess the cost and feasibility of running the application in a production environment.
+
+## Assumptions
+
+The Performance Analyzer tool generates forecasts based on the following assumptions:
+
+- For service-based applications the overhead of non-I/O operations is negligible.
+- The service/integration being evaluated is allocated sufficient resources, and therefore, it is not a bottleneck.
```

Comment:
I think performance forecasts are given only for service type applications, so basically we can use the term service instead of application, and we need to remove integration app type

Reply:
@miyurud please confirm